app/api/resources/metadata.py

```diff
@@ -238,3 +239,21 @@ async def get_genome_dataset_attributes(
     except Exception as ex:
         logging.error(ex)
         return response_error_handler({"status": 500})
+
+@router.get("/genome")
```

Comment:
Didn't we say that since `genome` in other routes has a different shape, this route should be called something other than `genome`?